src/components/instructor/AssignmentsView.js

```diff
@@ -9,16 +11,83 @@ import {useLocation} from 'react-router-dom'
 // display a table with columns 
 // assignment id, title, dueDate and buttons to grade, edit, delete each assignment
 
-const AssignmentsView = (props) => {
+const [assignments, setAssignments] = useState([]);
+const [message, setMessage] = useState('');
 
-    const location = useLocation();
-    const {secNo, courseId, secId} = location.state;
-     
-    return(
-        <> 
-           <h3>Not implemented</h3>
-        </>
-    );
-}
+useEffect(() => {
+    fetchAssignments();
+}, []);
 
-export default AssignmentsView;
+const fetchAssignments = async () => {
+    try {
+        const response = await fetch(`${SERVER_URL}/sections/${secNo}/assignments`);
+        if (response.ok) {
+            const assignments = await response.json();
```

Comment:
I would change the variable here to `data` or something similar. The name `assignments` is already taken in the outside scope, I would guess that this would cause issues.